pkgs/development/libraries/composable_kernel/default.nix

```diff
@@ -14,108 +13,73 @@
 , gpuTargets ? [ ] # gpuTargets = [ "gfx803" "gfx900" "gfx1030" ... ]
 }:
 
-let
-  # This is now over 3GB, to allow hydra caching we separate it
-  ck = stdenv.mkDerivation (finalAttrs: {
-    pname = "composable_kernel";
-    version = "unstable-2023-01-16";
-
-    outputs = [
-      "out"
-    ] ++ lib.optionals buildTests [
-      "test"
-    ] ++ lib.optionals buildExamples [
-      "example"
-    ];
-
-    # ROCm 5.6 should release composable_kernel as stable with a tag in the future
-    src = fetchFromGitHub {
-      owner = "ROCmSoftwarePlatform";
-      repo = "composable_kernel";
-      rev = "80e05267417f948e4f7e63c0fe807106d9a0c0ef";
-      hash = "sha256-+c0E2UtlG/abweLwCWWjNHDO5ZvSIVKwwwettT9mqR4=";
-    };
-
-    nativeBuildInputs = [
-      cmake
-      rocm-cmake
-      hip
-      clang-tools-extra
-    ];
-
-    buildInputs = [
-      openmp
-    ];
-
-    cmakeFlags = [
-      "-DCMAKE_C_COMPILER=hipcc"
-      "-DCMAKE_CXX_COMPILER=hipcc"
-    ] ++ lib.optionals (gpuTargets != [ ]) [
-      "-DGPU_TARGETS=${lib.concatStringsSep ";" gpuTargets}"
-      "-DAMDGPU_TARGETS=${lib.concatStringsSep ";" gpuTargets}"
```

Comment:
I don't think this is needed?

Reply:
We include this in case someone only wants to build for one GPU target, same for most other ROCm derivations.

Reply:
Oh yep, but I think `GPU_TARGETS` should already cover that for this project, right?

Reply:
IIRC, AMDGPU_TARGETS is just to ensure it doesn't mess up.
I remember having some sort of problem.

Reply:
Ok! Will add it back and rebase to fix the merge conflict